packages/base/any/kernels/5.10-lts/patches/0024-kernel-5-10-4-comphy-support-2500-BaseT.patch

```diff
@@ -0,0 +1,103 @@
+diff --git a/drivers/net/ethernet/marvell/mvpp2/mvpp2_main.c b/drivers/net/ethernet/marvell/mvpp2/mvpp2_main.c
+index cea886c5b..f53251c62 100644
```

Comment:
Please use the format created by `git format-patch` and add a commit message with a Signed-off-by line.